Pipfile

```diff
@@ -11,7 +11,7 @@ django = "==2.2.4"
 gunicorn = "*"
 django-compressor = "*"
 django-compressor-toolkit = "*"
-boto3 = "*"
+boto3 = "==1.9.215"
```

Comment:
@LindsayYoung Is this change related to the View Table or unrelated?

Reply:
I was having deploy issues when I was checking this, so I added that, I think it was probably the AWS region variable name change. We can git rid of this change if you want.

Reply:
Hmmmm, since it's not related to the view table feature, I can back it out. We can introduce this change in another PR if it's important to getting the deploy process working.

Reply:
Done